@navikt/core/react/src/stepper/Step.tsx

```diff
@@ -0,0 +1,61 @@
+import cl from "classnames";
+import React, { forwardRef, useContext } from "react";
+import { StepperContext } from ".";
+import { Label, OverridableComponent } from "..";
+
+export interface StepperStepProps
+  extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  /**
+   * Text content under indicator
+   */
+  children: React.ReactNode;
+  /**
+   * Handled by Stepper
+   */
+  index?: number;
+}
+
+export interface StepperStepType
+  extends OverridableComponent<StepperStepProps, HTMLButtonElement> {}
+
+const StepComponent: OverridableComponent<
+  StepperStepProps,
+  HTMLButtonElement
+> = forwardRef(
+  (
+    { className, children, as: Component = "button", index = 0, ...rest },
+    ref
+  ) => {
+    const context = useContext(StepperContext);
+
+    if (context === null) {
+      console.error(
+        "<StepIndicator.Step> has to be used within an <StepIndicator>"
+      );
+      return null;
+    }
+
+    return (
+      <Component
+        {...rest}
+        ref={ref}
+        className={cl("navds-stepper__step", className, {
+          "navds-stepper__step--active": context.activeStep === index,
+        })}
+        onClick={(e) => {
+          context.onStepChange(index);
+          rest?.onClick?.(e);
+        }}
+      >
+        <Label className="navds-stepper__step-number">
```

Comment:
```suggestion
        <Label className="navds-stepper__step-number" as="span">
```
Endrer `<p>` -> `<span>`